src/ui/elements/quarks/box/index.ts

```diff
@@ -0,0 +1,13 @@
+import { css } from 'styled-components';
+
+export const verticalBox = css`
```

Comment:
```suggestion
export const boxVertical = css`
```

Мне кажется, лучше именовать от общего к частному.
Общее тут будет box, а его частный случай это vertical. Есть разные боксы, этот конкретно vertical